### DISCUSSION TITLE
Wiki.js configure using SSH authentication

### DISCUSSION THREAD

**Discussion by lvp1138wikijs** [requarks/wiki#3318] [Error / Bug Report]
Hello,

Unable to configure wiki.js with SSH authentication. Getting the below error:
====
7T00:35:54.947Z [MASTER] warn: fatal: ambiguous argument 'origin/master': unknown revision o
Jan 26 16:35:54 lasvegas-nv-datacenter.serverpoint.com node[1039]: Use '--' to separate paths from revisions, like this:
Jan 26 16:35:54 lasvegas-nv-datacenter.serverpoint.com node[1039]: 'git [...] -- [...]'
======

1. Go to wiki admin area
2. Click on Storage
3. Scroll down to Git
4. Have entered all the settings for SSH authentication. While clicking "RUN" option in Force Sync, you will see the error. 

**Host Info (please complete the following information):**
 - OS: Ubuntu 16.04.4 
 - Wiki.js version: 2.5.170
 - Database engine: postgres 

Regards,
Tech

**Comment by NGPixel** [comment]
Did you save the config BEFORE running the Force Sync action?
